src/db/db.ts

```diff
@@ -1156,6 +1164,7 @@ export class Db {
     const resp = await this.#httpClient.executeCommand(command, {
       timeoutManager: this.#httpClient.tm.single('tableAdminTimeoutMs', options),
       keyspace: options?.keyspace,
+      extraLogInfo: options?.nameOnly ? 'name only' : 'with options',
```

Comment:
I'm not a big fan of this logging pattern, I think it would be better to just log the options as-is. Otherwise you need to document and users need to understand how to interpret "name only" vs "with options"